cn.lpkt.serverbox.app.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=ServerBox
+Exec=/app/AppRun
```

Comment:
```suggestion
Exec=/app/AppRun
```

install the executable to `/app/bin` then drop the path from here

Reply:
I think bbhtt meant:

```suggestion
Exec=AppRun
```